node/package.json

```diff
@@ -14,16 +14,16 @@
   "private": false,
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
-  "type": "module",
```

Comment:
This was causing some of the TS tooling to expect an `mjs` or `cjs` file. I tested using this as a dep and didn't seem to need this.